content/1.2/docs/quickstart/index.md

```diff
@@ -6,10 +6,10 @@ weight: 1
 
 ## Quickstart Guide
 
-This quickstart guide will help you install the Orchestrator using the helm chart and execute a sample workflow through the Red Hat Developer Hub orchestrator plugin UI.
+This quickstart guide will help you install the Orchestrator using the Helm-based operator and execute a sample workflow through the Red Hat Developer Hub orchestrator plugin UI.
```

Comment:
Do you think we should update also [this](https://github.com/parodos-dev/parodos-dev.github.io/blob/aa5a5c4b9575de013e8196e95eb289daa9d13000/content/1.2/docs/installation/_index.md?plain=1#L7) occurrence of "helm chart"?

Reply:
do you refer to the valina k8s? in that case it should remain helm chart, since this is the only way for k8s/kind/minikube to install the orchestrator.